src/components/Tab.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div v-if="isSelected" class="">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
```

Comment:
The component name should be multi-word, also for good practice add the Name of the component,